src/index.ts

```diff
@@ -164,6 +163,17 @@ const recommended = {
   [noImportNodeTestName]: 'error'
 } as const
 
+interface VitestPlugin extends Linter.Plugin {
+  meta: {
+    name: string
+    version: string
+  }
+  rules: Record<string, RuleModule<any, any>>
+  // TODO: use classic type for config
+  configs: Record<string, any>
+  environments?: Record<string, any>
```

Comment:
is there any way we can typecheck this correctly? I don't think using any here is the correct way to do it.